x-pack/plugins/ml/public/application/model_management/test_models/models/inference_base.ts

```diff
@@ -283,7 +290,11 @@ export abstract class InferenceBase<TInferResponse> {
     try {
       this.setRunning();
       const inputText = this.inputText$.getValue()[0];
-      const inferenceConfig = getInferenceConfig();
+      const pipelineForCreation = this.getPipelineForCreation();
+      let inferenceConfig = getInferenceConfig();
+      if (pipelineForCreation?.processors && pipelineForCreation.processors.length) {
```

Comment:
This _shouldn't_ be possible since we don't allow the user to safe edits to the processor without inference config being defined but went ahead and added the extra check as extra confirmation.
41be898f864b7f6e22af9eb426e374fed3cfb2a8